packages/combobox/src/Combobox.ts

```diff
@@ -384,12 +385,7 @@ export class Combobox extends Textfield {
                     ? 'focus-visible is-keyboardFocused'
                     : ''}"
                 ?focused=${this.focused}
-            >
-                <sp-icon-chevron100
-                    slot="icon"
-                    class="spectrum-UIIcon-ChevronDown100 icon"
-                ></sp-icon-chevron100>
```

Comment:
Removed this because the picker button handles the chevron sizing.

Reply:
Awesome!